local_settings.py.tmpl

```diff
@@ -112,6 +112,12 @@ ALLOWED_DOCUMENT_TYPES = [
 ]
 MAX_DOCUMENT_SIZE = 100
 
+STANDALONE_APPS = (
```

Comment:
Like the other PR: do not replicate in `INSTALLED_APPS` the `STANDALONE_APPS` list but concatenate to it like:
https://github.com/gem/oq-platform/blob/master/openquakeplatform/openquakeplatform/local_settings.py.template#L292